### ISSUE TITLE
.htaccess gets created but is empty

### ISSUE THREAD

**Issue by simonfranzen** [AndreasFaust/gatsby-plugin-htaccess#1]
Hey, thank you for the nice plugin. But I had one issue. When it generates the .htaccess the overall build script finishes before the .htaccess file is getting filled with content.

https://github.com/AndreasFaust/gatsby-plugin-htaccess/blob/ba9b33df2abf1efb10a16d80f596ba51a42d275f/gatsby-node.js#L39

What about an `await` before the writeFile statement?

What currently works for me:

```
return fs
      .ensureFile(htPath)
      .then(async () => {
          await fs.writeFile(htPath, htContent)
          await fs.writeFile(htStaticPath, htStaticContent)
      })
      .catch(e => console.error('onPostBuild error #hq0Kxa', JSON.stringify(e)))
```

But I am not sure about this await/sync in the Promise chain. Can you help?

Gatsby v 2.0

**Comment by simonfranzen** [comment]
When I put the plugin higher up in the gatsby-config it works, because it has a bit more time I thiink

**Comment by AndreasFaust** [comment]
Thank you for the hint! 
Maybe we should get rid of the whole promise chain and consistently work with async/await.
I will integrate this fix ASAP.

**Comment by simonfranzen** [comment]
Thank you! Until then I will put it into the beginning of the gatsby-config. Since I did this I dont have problems anymore

**Comment by AndreasFaust** [comment]
I fixed it and released version 1.0.9.
